helpers/createApiClient.js

```diff
@@ -0,0 +1,65 @@
+import feathers from '@feathersjs/feathers'
+import socketio from '@feathersjs/socketio-client'
+import io from 'socket.io-client'
+import authentication from '@feathersjs/authentication-client'
+import urlHelper from '~/helpers/urls'
+import Cookie from 'cookie-universal'
+
+const authKey = 'feathers-jwt'
+const endpoint = urlHelper.buildEndpointURL(process.env.API_HOST, { port: process.env.API_PORT })
```

Comment:
@appinteractive help! After merging/rebasing on `fix-docker-setup` I'm facing the problem again, that `process.env` is set at build time. I cannot see a solution here. :crying_cat_face:

Reply:
@roschaefer 
Try adding the variables you need at runtime from env in the webpack.EnvironmentPlugin like so (the second parameter is the 'default' value in case the env variable isn't set):
```
diff --git a/nuxt.config.js b/nuxt.config.js
index 6e88623..e4be132 100644
--- a/nuxt.config.js
+++ b/nuxt.config.js
@@ -1,5 +1,6 @@
 require('dotenv').config()
 const path = require('path')
+const webpack = require('webpack')
 
 module.exports = {
   env: {
@@ -90,6 +91,7 @@ module.exports = {
       // Mapbox-gl throws error after being uglified
       // https://github.com/mapbox/mapbox-gl-js/issues/4359
       config.module.noParse = /(mapbox-gl)\.js$/
+      config.plugins.unshift(new webpack.EnvironmentPlugin({API_HOST: 'localhost'}))
     }
   },
   plugins: [
```